recipes/necat/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -ex
+
+export CFLAGS="$CFLAGS  -I$PREFIX/include"
+export CXXFLAGS="$CXXFLAGS  -I$PREFIX/include"
+export LDFLAGS="$LDFLAGS -L$PREFIX/lib"
+
+# build
+cd src
+make CFLAGS+="${CFLAGS}" LDFLAGS+="${LDFLAGS}" CXXFLAGS+="${CXXFLAGS}"
+cd ..
+
+# create and populate binary file
+mkdir -p $PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM/bin/
```

Comment:
The FSH standard requires the path to be `/usr/share`

Reply:
Her I follow the [java recommendation](https://bioconda.github.io/contributor/guidelines.html?highlight=java#java) it's maybe not a good idea.

Reply:
ah, I see. Never mind then.